openvpn-formula/openvpn-formula.changes

```diff
@@ -5,6 +5,6 @@ Tue Aug 11 12:37:26 UTC 2020 - Alexander Graul <alexander.graul@suse.com>
 - Add hint that ssl certs must be on system (bsc#1172279)
 
 -------------------------------------------------------------------
-Mon Jul  6 11:01:18 UTC 2020 - Alexander Graul <alexander.graul@suse.com>
+Mon Apr 27 14:35:26 UTC 2020 - Alexander Graul <agraul@suse.com>
 
-- Initial Package v0.1
+- Inital Package v0.1
```

Comment:
I know this happens because you copied the file from OBS, but in this case autobuild will tolerate a change.

You can fix the typo so it says `Initial`

Reply:
Ok, so only updating the date was the problem? That makes sense, somehow I did not think about that when we did it.

Reply:
Right. Autobuild doesn't complain if we fix a typo, or if we add a missing bsc.

But they will complain (reject the submissions) if we rewrite/remove the date headers, remove bsc IDs, remove entries...

JFYI there's one big exception to this rule: when there's a new SUSE Manager version, certain packages such a salt will get a major version bump will not have incremental changelogs. In this case autobuild accepts rewrite as long as no bsc IDs are removed, and as long as the changelog information about the patches is still correct.

But this doesn't happen for the formulas (at least for now) as we use exactly the same package in all codestreams.

Reply:
thanks for clarifying! I'll send new submissions your way shortly (only need to merge,  adjust tag and run my obs service again)